vendor.conf

```diff
@@ -23,11 +23,11 @@ github.com/jmespath/go-jmespath bd40a432e4c76585ef6b72d3fd96fb9b6dc7b68d
 github.com/matttproud/golang_protobuf_extensions c12348ce28de40eed0136aa2b644d0ee0650e56c
 github.com/miekg/dns 271c58e0c14f552178ea321a545ff9af38930f39
 github.com/mitchellh/mapstructure 482a9fd5fa83e8c4e7817413b80f3eb8feec03ef
-github.com/ncw/swift b964f2ca856aac39885e258ad25aec08d5f64ee6
 github.com/prometheus/client_golang c332b6f63c0658a65eca15c0e5247ded801cf564
 github.com/prometheus/client_model 99fa1f4be8e564e8a6b613da7fa6f46c9edafc6c
 github.com/prometheus/common 89604d197083d4781071d3c65855d24ecfb0a563
 github.com/prometheus/procfs cb4147076ac75738c9a7d279075a253c0cc5acbd
+github.com/ncw/swift c95c6e5c2d1a3d37fc44c8c6dc9e231c7500667d
```

Comment:
Thanks for the change @mlmhl! Any chance you can go with one of the [official release commits](https://github.com/ncw/swift/releases) instead? Also, no reason to move the line. Let's try to keep vendoring in alphabetical order. Apologies for adding nits after so much time.

Reply:
@mlmhl Following up on this the latest official release tag is v.1.0.39 which refers to commit b2a7479cf26fa841ff90dd932d0221cb5c50782d

Reply:
Thanks @cquon ! Sorry for the late, already updated to b2a7479cf26fa841ff90dd932d0221cb5c50782d.